autopush/endpoint.py

```diff
@@ -335,8 +335,11 @@ def _overload_err(self, fail):
     def _router_fail_err(self, fail):
         """errBack for router failures"""
         fail.trap(RouterException)
-        log.err(fail, **self._client_info())
         exc = fail.value
+        if not str(exc.status_code).startswith('2'):
+            # Only log non-2XX responses, we sometimes have to throw one to
```

Comment:
Heh, not that it matters for this, but it's 3x slower than numeric range checking. (Yeah, I was curious so i ran a test)